File: SDKs/Aspose.Words_Cloud_SDK_for_Java/src/main/java/com/aspose/words/model/DocumentProperties.java
package com.aspose.words.model;

import java.util.*;
import com.aspose.words.model.Link;
public class DocumentProperties {
  private List<DocumentProperty> List = new ArrayList<DocumentProperty>();
  private Link link = null;
  /**
	 * getList
	 * Gets List<DocumentProperty>
	 * @return List
	 */
  public List<DocumentProperty> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<DocumentProperty>
	 * @param List List<DocumentProperty>
	 */
  public void setList(List<DocumentProperty> List) {
    this.List = List;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentProperties {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/DocumentProperties.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class DocumentProperties {
    public List<DocumentProperty> List { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DocumentProperties {\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
